src/main/java/com/easypost/model/AddressVerification.java

```diff
@@ -8,4 +8,31 @@ public final class AddressVerification {
     private Boolean success;
     private List<Error> errors;
     private AddressDetail details;
+
+    /**
+     * Set the success of this address verification object.
+     *
+     * @param success The success message.
+     */
+    void setSuccess(final boolean success) {
+        this.success = success;
+    }
+
+    /**
+     * Set the errors of this address verification object.
+     *
+     * @param errors The errors.
+     */
+    void setErrors(final List<Error> errors) {
+        this.errors = errors;
+    }
+
+    /**
+     * Set the details of this address verification object.
+     *
+     * @param details The details.
+     */
+    void setDetails(final AddressDetail details) {
+        this.details = details;
+    }
```

Comment:
My concern/question about that, would that make the publicly-accessible setters or internal only?

Reply:
I just found out that you can define it in Lombok by using `AccessLevel.PRIVATE`: https://projectlombok.org/features/experimental/FieldDefaults

Reply:
Doesn't look like that applies to setter/getters. I'm going to keep it as is since that mimics the explicit setters we have in the `Error` class as well.